src/main/java/io/github/hapjava/services/Service.java

```diff
@@ -29,4 +29,6 @@
    *     ########-####-####-####-############.
    */
   String getType();
+
```

Comment:
please add
```suggestion
void addLinkedService(Service s);

```

Reply:
Characteristics are handled just the same as LinkedServices in the `Service` interface and `AbstractServiceImpl`. And so I didn't add `addLinkedService` to the `Service` interface, but only to the abstract class, which is just how it's done for Characteristics as well.